docs/guides/api.en.md

```diff
@@ -86,6 +89,63 @@
 
 ---
 
+## Events
+
+??? example "AddNewEvent `method`"
+    <h3>Summary</h3>
+    Adds a new event to a gameobject.
+    <h3>Examples</h3>
+    ```c#
+    // Example 1 : Create Event from string, with object name
+    Msl.AddNewEvent("o_myobject", "scr_actionsLogUpdate(\"hello world\")", EventType.Create, 0);
+
+    // Example 2 : Other Event from file, with object name
+    Msl.AddNewEvent("o_anotherobject", ModFiles.GetCode("myScript.gml"), EventType.Other, 24);
+
+    // Example 3 : Create Event from string, with object as variable
+    Msl.AddNewEvent(myVarContainingAGameObject, "scr_actionsLogUpdate(\"hello world\")", EventType.Create, 0);
+    ```
+    <h3>Arguments</h3>
+
+    | **Type** | **Name** | **Description** |
+    | :---: | :---: | :--- |
+    | `String` | `gameObject` | The name of the gameobject to add the event to. |
```

Comment:
how can we display the different signature (ie here we can either use a string or a UnderTaleGameObject as said in the previous example)

Reply:
Check the `Summary` section of the `Patching : Stage 2 (Matching)`, that's how I displayed it before